apps/snitch_core/test/data/schema/review/review_test.exs

```diff
@@ -0,0 +1,71 @@
+defmodule Snitch.Data.Schema.ReviewTest do
+  use ExUnit.Case, async: true
+  use Snitch.DataCase
+
+  import Snitch.Factory
+
+  alias Snitch.Data.Schema.Review
+
+  setup :review_params
+
+  describe "create_changeset/2" do
+    test "returns a valid changeset", %{params: params} do
+      changeset = Review.create_changeset(%Review{}, params)
+      assert changeset.valid?
+    end
+
+    test "returns invalid changeset" do
+      changeset = Review.create_changeset(%Review{}, %{})
+
+      assert %{
+               rating_option_vote: ["can't be blank"],
+               description: ["can't be blank"],
+               user_id: ["can't be blank"],
+               name: ["can't be blank"]
+             } == errors_on(changeset)
+    end
+
+    test "fails for duplicate user_id", %{params: params} do
```

Comment:
Test statement not correct, you have mentioned duplicate id but you are testing for non-existent one.